src/view/QuestionsList.js

```diff
@@ -0,0 +1,49 @@
+
+import React from "react";
+
+const QuestionsList = ({questions, onCreateQuestion, onViewTags, onFilterByTag, onFilterByTitle}) => (
+    <div>
+        <h2>Questions</h2>
+        <br/> <br/> 
+        <label>Search: </label>
+            <input //value={search} onSearch={ e =>onSearch("userId",e.target.value)}
+            />
+        <br />
+        <button onClick = {onFilterByTag}>Filter questions by tag</button>
+        <br/> 
+        <button onClick = {onFilterByTitle}>Filter questions by title</button>
+        <br/> <br/> 
+        <table border = "1">
+            <thead>
+                <tr>
+                    <th>User</th>
+                    <th>Title</th>
+                    <th>Text</th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody>
+            {
+                questions.map((question, index) => (
+                    <tr key = {index}>
```

Comment:
As you have this table duplicated between the QuestionList and the FilterBy... components, I would suggest that you extract a dumb component out of the table.

Reply:
Yes, I will try to do this and use that dumb component every time when I have to display a list of questions.